### DISCUSSION TITLE
XML highlighting suddenly changed.

### DISCUSSION THREAD

**Discussion by ajoino** [HiPhish/rainbow-delimiters.nvim#87] [General]
Hi, I'm running a custom xml query (i.e. mostly copied the html example) such that only the angle brackets were highlighted and not the tags. The query in question is
```scheme
(XMLDecl "<?" @delimiter "?>" @delimiter @sentinel) @container
(element 
  ((STag
     "<" @delimiter ">" @delimiter)
   ; (content)
   (ETag
     "</" @delimiter ">" @delimiter @sentinel))) @container

(element
  (EmptyElemTag
    "<" @delimiter "/>" @delimiter @sentinel)) @container
```

However, when I updated my plugins recently it changed so that the tag is also highlighted, and try as I might I couldn't figure out how to not make it highlighted. Did something change in this plugin or did something happen with nvim-treesitter?

I'm opening this as a discussion as I'm not sure if it's a bug or pebcak.

As a side point, would you like me to submit a PR with these highlights? Apart from the tag-highlighting they work pretty damn well IMO.

**Comment by ajoino** [comment]
Huh, I think I figured out the problem, you added XML highlights two weeks ago!
I just kinda assumed nothing would happen on that point, so I didn't check before.

I'll go read the docs to see how I can override the default highlights, if I come to an answer I'll close this and if I don't I'll come back.

**Comment by HiPhish** [comment]
Yes, I did add XML support. You should be able to override the query by adding your own query to `after/queries/xml/rainbow-delimiters.scm`. If it is in the `after` directory it should override the included one. Alternatively you can give it a different name like `rainbow-angle-brackets.scm` and then choose this name in your configuration:

```vim
let g:rainbow_delimiters = {'query': {'xml': 'rainbow-angle-brackets'}}
```
